uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Validator/Date.php

```diff
@@ -22,7 +24,10 @@ class CRM_HRLeaveAndAbsences_Validator_Date {
   public static function isValid($date, $format = 'Y-m-d')
   {
     $dateTime = DateTime::createFromFormat($format, $date);
-
+    if (!$dateTime && $format != 'YmdHis') {
+      $format = 'YmdHis';
+      $dateTime = DateTime::createFromFormat($format, $date);
+    }
```

Comment:
Another thing. This change will certainly break things on the AbsencePeriod BAO validation and the AbsencePeriod form. It might be good to run their tests too and check what will break